src/fu-util.c

```diff
@@ -2510,9 +2510,9 @@ main (int argc, char *argv[])
 #ifdef HAVE_SYSTEMD
 	/* make sure the correct daemon is in use */
 	if (!fu_util_using_correct_daemon (&error)) {
-		g_printerr ("Daemon verification failed: %s\n",
+		g_warning ("Daemon verification failed: %s\n",
 			    error->message);
-		return EXIT_FAILURE;
+		g_clear_error (&error);
```

Comment:
FYI, I'd be fine with a hard error rather than a warning.

Reply:
OK, I made some adjustments for this that I think should still let the test suite be happy.